holoviews/plotting/plotly/renderer.py

```diff
@@ -54,7 +55,7 @@ def __call__(self, obj, fmt='html', divuuid=None):
 
         if isinstance(plot, tuple(self.widgets.values())):
             return plot(), mime_types
-        elif fmt == 'html':
+        elif fmt in ('html', 'png', 'svg'):
             return self._figure_data(plot, divuuid=divuuid), mime_types
```

Comment:
I got an error trying `hv.save` with this, and I *think* the issue is that we need to pass `fmt` into `self._figure_data` here. Otherwise it will return the script as a string

Reply:
Yep, thanks noticed that too. Fixed in another PR.